admiral/pkg/controller/admiral/shard.go

```diff
@@ -149,12 +149,12 @@ func NewShardController(stopCh <-chan struct{}, handler ShardHandler, configPath
 	if err != nil {
 		return nil, fmt.Errorf("failed to create shard controller crd client: %v", err)
 	}
-	//TODO: should not be hardcoded, fetch actual expected operator and shard identities from env variables
-	//labelOptions := informers.WithTweakListOptions(func(opts *metav1.ListOptions) {
-	//	opts.LabelSelector = "admiral.io/operatorIdentity=operatorIdentity, admiral.io/shardIdentity=dev"
-	//})
-	//informerFactory := informers.NewSharedInformerFactoryWithOptions(shardController.K8sClient, resyncPeriod, labelOptions)
-	informerFactory := informers.NewSharedInformerFactoryWithOptions(shardController.K8sClient, resyncPeriod)
+	labelOptions := informers.WithTweakListOptions(func(opts *metav1.ListOptions) {
+		opIdLabel, opIdValue := common.GetOperatorIdentityLabelKeyValueSet()
```

Comment:
Can we use a constant label key, and provide ability to configure different values. That way we will standardize the key amongst all admiral operators.